source/core/Pipeline.cpp

```diff
@@ -296,6 +296,15 @@ ErrorCode Pipeline::encode(bool isStatic, bool supportDebug) {
     }
 #endif
 
+    // cached
+    command_cached.clear();
+    for (auto& info : mInfo) {
```

Comment:
Merge command in info is to reduce the compute time for Geometry::encode.

Reply:
in some model, mInfo->executeBuffer->command frequently empty